internal/service/project/model_project.go

```diff
@@ -202,3 +206,32 @@ func UpdateProjectBool(plan, state types.Bool, setting **bool) bool {
 	}
 	return false
 }
+
+func NewTFTags(tags *[]admin.ResourceTag) types.Map {
```

Comment:
i guess conversion FlattenTags & ExpandTagsFromSetSchema can't be used because different types.  
Don't know if we should extract this function or better wait until it's needed in another resource

Reply:
Good point, could also apply for the schema definition of tags.

Reply:
I would suggest to wait until we have a need (YAGNI) - unless we do have a need now :D